### PR TITLE
Show amp app banner in opera

### PR DESCRIPTION
**Pull request by muxin**
Fix https://github.com/ampproject/amphtml/issues/10415

### CODE REVIEW

File: src/service/platform-impl.js [hunk 1]
```diff
@@ -74,6 +74,14 @@ export class Platform {
   }
 
   /**
+   * Whether the current browser is an Opera browser.
+   * @return {boolean}
+   */
+  isOpera() {
```
Comment (aghassemi) [inline src/service/platform-impl.js]: add test to `test-platform.js`

**Comment by muxin** [comment]
@aghassemi PTAL. Manual tested on Android/IOS devices with Opera.

File: src/service/platform-impl.js [hunk 2]
```diff
@@ -94,7 +104,8 @@ export class Platform {
    * @return {boolean}
    */
   isWebKit() {
-    return /WebKit/i.test(this.navigator_.userAgent) && !this.isEdge();
+    return /WebKit/i.test(this.navigator_.userAgent) && !this.isEdge()
```
Comment (aghassemi) [inline src/service/platform-impl.js]: hmm, Opera is Webkit, any particular reason this was added?

Comment (muxin) [reply src/service/platform-impl.js]: Oh, I'll delete this.

**Comment by muxin** [comment]
@aghassemi PTAL

**Pull request merged**